client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html

```diff
@@ -709,8 +709,8 @@
                         <label class="control-label"
                                data-translate="telephony_service_contact_status">
                         </label>
-                        <p class="form-control-static"
-                           data-ng-bind="'telephony_service_contact_sync' + (ServiceContactCtrl.directoryForm.status === 'todo' ? '_pending' : '') | translate">
+                        <p class="form-control-static" data-ng-bind=":: ServiceContactCtrl.getStatusLabel()">
+                            {{ServiceContactCtrl.getStatusLabel()}}
```

Comment:
You don't need mustache templating with `ng-bind`

Reply:
What Mary means is that mustache templating is worse for performances than directives:

```suggestion
                            <span data-ng-bind="ServiceContactCtrl.getStatusLabel()}"></span>
```

Reply:
Please pay attention that the ngBind directive is already applied to the `<p>` element.

Could you please remove the extra `span` element?

Thanks!

Reply:
> What Mary means is that mustache templating is worse for performances than directives:

@FredericEspiau I meant that having both is not necessary. By the way it's Marie 😉

Reply:
> > What Mary means is that mustache templating is worse for performances than directives:
> 
> @FredericEspiau I meant that having both is not necessary. By the way it's Marie 😉

What Frédérik means is that he's sorry he got your name wrong